### ISSUE TITLE
Audio output forward - alarm audio is not being forwarded

### ISSUE THREAD

**Issue by Neurovert** [Genymobile/scrcpy#4315]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Linux Manjaro Uranus 23.0.0
 - scrcpy version: 2.1.1
 - installation method: Manjaro Official Repo
 - device model: Google Pixel 6**a**
 - Android version: 13

**Describe the bug**
When an alarm sounds, it sounds only on the phone, **it is NOT forwarded via Scrcpy** eventhough the feature is enabled.
In other words - I can't hear the alarm ringing on PC with Scrcpy.
Media audio forwarding works just fine with scrcpy.

I'm guessing that Android native audio forwarding is only for media volume? 
Could we somehow add the alarms too? 
When I'm on my noise cancelling headphones connected to the PC (not the Android phone) and use Scrcpy, I annoy the people around me because they hear the alarms much sooner than I do 😅

**Comment by OldGuyInTheClub** [comment]
I have this same question and am asking here according to the terms of use regarding researching existing issues.

scrcpy works great for me for apps like NewPipe, VLC, and similar.  I can use them wirelessly from my Macbook.  I would like to be able to set an alarm or a timer on the Clock app and hear the sound but these sounds only play on the device.  Is it possible to have these sounds play on the remote computer?

**scrcpy configuration:**
scrcpy 3.3.1 <https://github.com/Genymobile/scrcpy>

Dependencies (compiled / linked):
 - SDL: 2.32.8 / 2.32.8
 - libavcodec: 61.19.101 / 61.19.101
 - libavformat: 61.7.100 / 61.7.100
 - libavutil: 59.39.100 / 59.39.100
 - libusb: - / 1.0.29

**Invocation:**
/opt/homebrew/bin/scrcpy --video-codec=h265 --max-size=1920 --max-fps=60  --keyboard=uhid --serial 192.168.68.105

- Setting --audio-source=output does not disable audio on the device as expected and the non-media sounds don't get sent to the Mac

**Mac configuration:**
Apple M3 Pro
Sequoia 15.6.1

**Phone configuration:**
Samsung Galaxy S10+
Android 12
OneUI 4.1

**Due diligence:**
Github issues, scrcpy documentation, Kagi searches including Fediverse, Programming, and Forums lenses.

Thanks,

The Old Guy In The Club
